mods/ts/rules/defaults.yaml

```diff
@@ -382,6 +382,8 @@
 		ReferencePoint: Bottom, Right
 		RequiresCondition: hospitalheal
 	RevealOnFire:
+	RejectsOrders:
```

Comment:
Infact, I'd prefer if we could allow multiple `RejectOrders` traits on actors along with that `Reject` field, would be helpful for modders.